spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
I've always felt tests like this are tricky, because they fall right in that grey area of "are we testing Rails, or are we testing our model's behavior that happens to be implemented via a Rails call"

In this particular case, it is completely testing how has_many works, which is not necessary.  In other words, this test could be completely written as the following, which seems kind of useless.

```ruby
expect(described_class.reflect_on_association(:authentications)).to be_kind_of(ActiveRecord::Reflection::HasManyReflection)
```

On the other hand, from a pure "business logic" point of view, I can see that you might want to test that we have connected these two things together and they work properly, but I think that's where slightly higher level integration tests might come into play.  So when you act on something that uses both of these component parts that in turn tests the existence of the relation.